### PR TITLE
This would have the CLI handle the commits and push so we can make

### PR DESCRIPTION
**Pull request by CJ-Wright**
the CI scripts  a bit more streamlined

Attn: @justcalamari @mariusvniekerk @scopatz

### CODE REVIEW

**Comment by CJ-Wright** [comment]
(Note that if this goes in I'll update the circle config accordingly)

File: conda_forge_tick/cli.xsh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import time
+
+from doctr.travis import run_command_hiding_token as doctr_run
+
+from .all_feedstocks import main as main0
+from .make_graph import main as main1
+from .update_upstream_versions import main as main2
+from .auto_tick import main as main3
+
+int_script_dict = {0: main0, 1: main1, 2: main2, 3: main3}
```
Comment (scopatz) [inline conda_forge_tick/cli.xsh]: Also known as a list :) `[main0, main1, ...]`

**Comment by scopatz** [comment]
Thanks @CJ-Wright!

File: conda_forge_tick/cli.xsh (hunk 1 above)
Comment (CJ-Wright) [reply conda_forge_tick/cli.xsh]: We might come up with more descriptive names at some point, you know if we got around to it.

Comment (scopatz) [reply conda_forge_tick/cli.xsh]: :+1:

**Pull request merged**